CHANGES.md

```diff
@@ -2,7 +2,7 @@
 
 ### Notable changes
 
-* Updated to [Marathon 1.6.564](https://github.com/mesosphere/marathon/tree/3fa693b32).
+* Updated to [Marathon 1.6.567](https://github.com/mesosphere/marathon/tree/2d8b3e438ffcc536ccf8b1ea9cb0b39bb3ef4e10).
```

Comment:
Thank you!

Reply:
But I think this `## DC/OS 1.11-dev` section needs a lot of work. The last section is titled 1.11.6 and we did not account for 1.11.7 in this file.